Test/Utils/ContrailUtils.ps1

```diff
@@ -142,6 +142,24 @@ function Remove-ContrailVirtualNetwork {
     Invoke-RestMethod -Uri $NetworkUrl -Headers @{"X-Auth-Token" = $AuthToken} -Method Delete | Out-Null
 }
 
+function Get-ContrailVirtualNetworkPorts {
```

Comment:
Please consider not putting more stuf into ContrailUtils file

Reply:
IDEA: These seem like implementations of methods in ContrailNetwokManager, so maybe create ContrailNetworkManager subdir, move ContrailnetworkManager.ps1 there, and try to split this ContrailUtils.ps1 file as well? I'm not sure

Reply:
As discussed IRL, in this PR I have created a subdir : https://github.com/Juniper/contrail-windows-ci/pull/277

Maybe we should add files like "Networks.ps1", "FloatingIP.ps1" etc there